research.md

```
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Research
+subtitle: Understanding microbial communities through systems biology
+permalink: /research/
+---
+
+# Research
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)